openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -523,11 +508,11 @@ OpenAssessment.ResponseView.prototype = {
     fileUpload: function() {
         var view = this;
         var fileUpload = $(".file__upload");
-        fileUpload.addClass("is--disabled");
+        fileUpload.prop('disabled', true);
 
         var handleError = function(errMsg) {
             view.baseView.toggleActionError('upload', errMsg);
-            fileUpload.removeClass("is--disabled");
+            fileUpload.prop('disabled', false);
```

Comment:
@staubina I can't remember exactly how you interacted with "disabled" in your recent PR. Do you think something should be updated there to reflect the documentation above?